### DISCUSSION TITLE
Running GPTH second time - still not working

### DISCUSSION THREAD

**Discussion by Amotsoko** [TheLastGimbus/GooglePhotosTakeoutHelper#256] [I NEED HELP]
Dear experts

I wrote a question here a couple of days ago (#254) regarding using GPTH again after making a mistake the first time.
The advice I got was that I cannot use it a second time in the same folder and that I should unzip all the files again.
I deleted all the files (zipped and unzipped), downloaded them again and unzipped them again in the same drive but not the same folder. However, I am still getting the exact same error:
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/50144878/e0c35193-72e6-4f51-a504-e3ee864f9416
What else can be done?
Thanks
Amots

**Comment by tbellrobinson** [comment]
Same boat. I've used this tool before, probably in version 2. Need to use it again but getting the same issue as you. Nothing I seem to do makes any change.

-downloaded from takeout
-extracted with 7zip and with windows built in
-selected source and destination
-same error every time
-i've also re-extracted as he mentions, doesn't change
-tried running in cmd only, no luck

Anyone have any other solutions?

edit again - used version 2.3.0 and it worked, but failed to insert exif on 631 out of 2000, even though most have the .json file with them.

**Comment by TheLastGimbus** [comment]
Okay okay, @Amotsoko and @tbellrobinson - the error says it did not find any photos - right now it seems that gpth is fully working for 99% of users so it's proooobably a problem with your selections etc - or, you just have a veeeery weird takeout

> i did not get any answers from you or from Synology.

Wait, you're running on Synology...? This is a completely different thing. You will have to run with cmd-only etc

> used version 2.3.0

Do NOT use version 2.x :pray: - it's veeeery old, and very buggy. It may lose some of your photos

How did you take your takeout? All folders, or just albums, or just years? Did you export them nicely to *one* folder as in visual instructions in readme? Do you actually have photos inside those folders? Show me the screenshots of those folders, their hierarchy etc

**Reply by bypsix** [reply]
I have the same error, maybe because the title is in Russian?
The path to the photo, for example, E:\ФотоВидеоАрхив\2022_\Takeout\Google Фото\2022\_20221221_195418.JPG
![7](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/18091452/db1f714f-b05c-44e9-aeac-b4cdecb2628e)

**Comment by badgandalf** [comment]
I dowloaded the newest version and got the exact same error on Windows 11. Any idea how to solve it? Or why does it refer to D: when I'm pointing to directories in C:? Thanks

**Comment by bypsix** [comment]
I understood the reason, the folder structure is strictly for the English version of Google Photos and should be in this template: **Takeout\Google Photos\Photos from 2022**
Тогда всё работает прекрасно!
![8](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/18091452/e18184b0-818b-400b-bf22-b39d9959ac63)

**Reply by badgandalf** [reply]
So, what would be the solution then?

**Reply by bypsix** [reply]
organize your folders to similarly Takeout\Google Photos\Photos from 2022

**Comment by peternijssen** [comment]
I've got a similar problem like the people before me. Running on OSX.

The directory structure I get back From Google is:
- Takeout
   -  Google Foto_s
       - Album A
       - Album B
       - Album C

```
Okay, running... searching for everything in input folder...
...oh :(
...
I couldn't find any D: reasons for this may be:
  - your Takeout doesn't have any "year folders" -
    visit https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper
    again and request new, correct Takeout
After fixing this, go ahead and try again :)
```

Can't figure out why it doesn't work :( 
Tried all different layers as input.
No issues with the 2.X version tool, but I would like to use V3 so I can keep the albums.

**Reply by TheLastGimbus** [reply]
Read the log message, again
